companies/outdoor-afro.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Outdoor Afro
+tags: ["outdoors"]
+blm_statements:
+  - url: https://twitter.com/OutdoorAfro/status/1266526972538245121
+    date_posted: 2020-05-29
+    type: Tweet
+    raw_text: >
+      We Stand for Our People
+      
```

Comment:
```suggestion

```

I think the linter wants the leading spaces removed from the blank lines.  Try that and see if it resolves the errors! ❤️

Reply:
Will do! These errors are driving me crazy LOL

Reply:
LOL i know... I am considering changing the data from YAML to JSON but worried it's too complicated now

Reply:
Let me know when you're planning to transform it! I'd love to help! @kelseyhuse30